### PR TITLE
Release 72.0.0

### PR DESCRIPTION
**Pull request by jpuri**
New release of PreferenceController to add another preference for security alerts.

### CODE REVIEW

**Review by mcmire — COMMENTED** [review]
Just one thing, but looks good otherwise.

File: packages/preferences-controller/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.3.0]
+### Added
+- Adding preference for security alerts ([#1589](https://github.com/MetaMask/core/pull/1589))
+- devDeps: update eslint packages ([#1498](https://github.com/MetaMask/core/pull/1498))
```
[the remark this answers is not in the document]
Comment (jpuri) [reply packages/preferences-controller/CHANGELOG.md:12]: Look at this commit: https://github.com/MetaMask/core/pull/1603/commits/294501a10710a5169af7d449c6cc1199360b4789

[the remark this answers is not in the document]
Comment (mcmire) [reply packages/preferences-controller/CHANGELOG.md:12]: Oh actually this change isn't necessary to include in the changelog — it's development-only. I can remove it.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**